src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/deleteroles/DeleteRoleAssignmentOrchestrator.java

```diff
@@ -281,5 +291,83 @@ private void updateRequestStatus(AssignmentRequest assignmentRequest, Status sta
 
     }
 
+    @SuppressWarnings("unchecked")
+    public ResponseEntity<Void> deleteRoleAssignmentByQuery(MultipleQueryRequest multipleQueryRequest) {
+
+        //1. create the request Object
+        if (CollectionUtils.isNotEmpty(multipleQueryRequest.getQueryRequests())) {
+            request = parseRequestService.prepareDeleteRequest("", "", "", "");
```

Comment:
The process should be set to "CCD" and reference="DeleteByQueryRequest"

Reply:
Done.